activerecord/test/models/entity.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class Entity < ActiveRecord::Base
+  belongs_to :entity, polymorphic: true
```

Comment:
Can you reuse existing model for testing belongs_to polymorphic association?

Like

```diff
diff --git a/activerecord/test/models/face.rb b/activerecord/test/models/face.rb
index 948435136d..54f079c429 100644
--- a/activerecord/test/models/face.rb
+++ b/activerecord/test/models/face.rb
@@ -13,3 +13,7 @@ class Face < ActiveRecord::Base
     man
   end
 end
+
+class NewFace < Face
+  belongs_to :new_face, polymorphic: true
+end
diff --git a/activerecord/test/schema/schema.rb b/activerecord/test/schema/schema.rb
index 7d008eecd5..4c01e35c56 100644
--- a/activerecord/test/schema/schema.rb
+++ b/activerecord/test/schema/schema.rb
@@ -951,6 +951,7 @@
     t.string  :poly_man_without_inverse_type
     t.integer :horrible_polymorphic_man_id
     t.string  :horrible_polymorphic_man_type
+    t.references :new_face, polymorphic: true, index: false
   end
 
   create_table :interests, force: true do |t|
```

Or

```diff
diff --git a/activerecord/test/models/sponsor.rb b/activerecord/test/models/sponsor.rb
index f190860fd1..18ff103ffe 100644
--- a/activerecord/test/models/sponsor.rb
+++ b/activerecord/test/models/sponsor.rb
@@ -3,6 +3,7 @@
 class Sponsor < ActiveRecord::Base
   belongs_to :sponsor_club, class_name: "Club", foreign_key: "club_id"
   belongs_to :sponsorable, polymorphic: true
+  belongs_to :sponsor, polymorphic: true
   belongs_to :thing, polymorphic: true, foreign_type: :sponsorable_type, foreign_key: :sponsorable_id
   belongs_to :sponsorable_with_conditions, -> { where name: "Ernie" }, polymorphic: true,
              foreign_type: "sponsorable_type", foreign_key: "sponsorable_id"
diff --git a/activerecord/test/schema/schema.rb b/activerecord/test/schema/schema.rb
index 7d008eecd5..e5fd881257 100644
--- a/activerecord/test/schema/schema.rb
+++ b/activerecord/test/schema/schema.rb
@@ -814,6 +814,7 @@
   create_table :sponsors, force: true do |t|
     t.integer :club_id
     t.references :sponsorable, polymorphic: true, index: false
+    t.references :sponsor, polymorphic: true, index: false
   end
 
   create_table :string_key_objects, id: false, force: true do |t|
```